packages/google_adsense/lib/src/adsense_web.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:web/web.dart' as web;
+import 'ad_unit_widget.dart';
+import 'ad_unit_widget_web.dart';
+
+/// Main class to work with the library
+class Adsense {
```

Comment:
I just noticed: Is it `Adsense` or `AdSense`? I think it's the latter, if so, the class names should be `AdSenseSomething`